docs/user_guide/examples/explanation_interpolation.md

```diff
@@ -1,4 +1,4 @@
-# Interpolator explanation
+# Interpolators
```

Comment:
Change this to `Interpolators Overview and API`, so that it is more explicit in the menu on the left?